content/r/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,215 @@
+# R Conditionals
+
+Conditional statements in R allow you to control the flow of your program based on certain conditions. You can execute different blocks of code depending on whether a condition is true or false.
+
+## If Statement
+
+The `if` statement is used to execute a block of code if a condition is true. The general syntax is:
+
+```r
+if (condition) {
+  # code to be executed if condition is true
+}
+```
+
+Here's an example:
+
+```r
+x <- 10
+
+if (x > 5) {
+  print("x is greater than 5")
+}
+
+```
+
+In this example, the code inside the `if` block will be executed because the condition `x > 5` is true.
+
+## If-Else Statement
+
+The `if-else` statement allows you to execute one block of code if a condition is true, and another block of code if the condition is false. The general syntax is:
+
+```r
+if (condition) {
+  # code to be executed if condition is true
+} else {
+  # code to be executed if condition is false
+}
+
+```
+
+Here's an example:
+
+```r
+x <- 3
+
+if (x > 5) {
+  print("x is greater than 5")
+} else {
+  print("x is less than or equal to 5")
+}
+
+```
+
+In this example, since the condition `x > 5` is false, the code inside the `else` block will be executed.
+
+## Comparison Operators
+
+Comparison operators in R allow you to compare values and produce logical results. Here are some commonly used comparison operators:
+
+- `<` (less than): Returns `TRUE` if the left operand is less than the right operand.
+    
+    Example:
+    
+    ```r
+    x <- 5
+    y <- 10
+    
+    if (x < y) {
+      print("x is less than y")
+    }
+    
+    ```
+    
+    Output: `x is less than y`
```

Comment:
Usually we prefer using a separate shell block for displaying the output; however, with entries such as this, with many examples, we allow for commented outputs within the same block.